src/cursor-doc/indent.ts

```diff
@@ -63,7 +63,12 @@ export function collectIndents(
   let lastLine = cursor.line;
   let lastIndent = 0;
   const indents: IndentInformation[] = [];
-  const rules = { ...config['cljfmt-options']['indents'], ...cljsLib.defaultIndents() };
+  const defaultIndents = cljsLib.defaultIndents();
+  const extraIndents = config['cljfmt-options']['indents'];
+  const defaultRules = Object.keys(defaultIndents).map((k) => [k, defaultIndents[k]]);
+  const extraRules = Object.keys(extraIndents).map((k) => [k, extraIndents[k]]);
+  const rules = [...extraRules, ...defaultRules];
```

Comment:
Could we not have just reversed the spread?

```typescript
const rules = { ...cljsLib.defaultIndents(), ...config['cljfmt-options']['indents'], };
```

Maybe I'm missing the core problem though. It seems that the user-provided indents should override the default where they intersect.